release_tools/changelog.py

```diff
@@ -211,13 +212,33 @@ def create_changelog_entry_content(title, category, author=None, issue=None,
     return content
 
 
-def write_changelog_entry(dirpath, title, content, overwrite=False):
-    """Store the contents of an entry in a file."""
-
+def validate_changelog_content(content, run_editor=True):
     if not content:
         msg = "Aborting due to empty entry content"
         raise click.ClickException(msg)
 
+    while True:
```

Comment:
I don't like this infinite loop here. I think it's better to exit the loop when the content is validated.